esp/src/src/Session.ts

```diff
@@ -0,0 +1,60 @@
+import * as cookie from "dojo/cookie";
+import * as xhr from "dojo/request/xhr";
+import * as ESPUtil from "./ESPUtil";
+import * as topic from "dojo/topic"
+
+const espTimeoutSeconds = cookie("ESPSessionTimeoutSeconds") || 600; // 10 minuntes?
+const IDLE_TIMEOUT = espTimeoutSeconds * 10;
+const SESSION_RESET_FREQ = 30 * 1000;
+
+let _prevReset = Date.now();
+let idleWatcher = new ESPUtil.IdleWatcher(IDLE_TIMEOUT);
+let monitorLockClick = new ESPUtil.MonitorLockClick();
+let sessionIsActive = espTimeoutSeconds;
+
+declare const dojoConfig;
```

Comment:
Minor:  by convention I tend to keep the global declares at the top of the file below the imports...